### PR TITLE
implement Event class

### PR DESCRIPTION
**Pull request by raultolive**

### CODE REVIEW

File: src/main/java/org/fjala/gugumber/salesforce/entities/Event.java [hunk 1]
```diff
@@ -12,93 +12,263 @@
...
+     * @return a string, this is "assignedTo" variable.
+     */
+    public String getAssignedToUser() {
         return assignedToUser;
     }
 
-    public void setAssignedToUser(User assignedToUser) {
+    /**
+     * Sets the value "assignedTo" variable.
+     */
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/entities/Event.java]: add param to the documentation

File: src/main/java/org/fjala/gugumber/salesforce/entities/Event.java [hunk 2]
```diff
@@ -12,93 +12,263 @@
...
+     * @return a string, this is "Subject" variable.
+     */
     public String getSubject() {
         return subject;
     }
 
-    public void setSubject(String subject) {
+    /**
+     * Sets the value "subject" variable.
+     */
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/entities/Event.java]: Add param to the documentation

File: src/main/java/org/fjala/gugumber/salesforce/entities/Event.java [hunk 3]
```diff
@@ -12,93 +12,263 @@
...
+    public void setNameContact(final String nameContact) {
+        this.nameContact = nameContact;
     }
 
-    public Account getRelatedToAccount() {
+    /**
+     * Gets the value assigned to "relatedToAccount" variable.
+     *
+     * @return a string, this is "relatedToAccount" variable.
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/entities/Event.java]: should be by example:
```suggestion
     * @return a the value of relatedToAccount as string.
```

File: src/main/java/org/fjala/gugumber/salesforce/entities/Event.java (hunk 2 above)
Comment (raultolive) [reply src/main/java/org/fjala/gugumber/salesforce/entities/Event.java]: okay

**Pull request merged**